conga-aem-definitions/src/main/roles/aem-dispatcher-ams.yaml

```diff
@@ -407,6 +407,9 @@ config:
     # Name of the farm
     farmName: publishfarm
 
+    # Controls the DispatcherPassError setting
```

Comment:
probably add a bit more comment whats possible settings here, e.g. 0=spool all errors to client, 1=pass all errors over to apache, code range=pass specific erros to apache

Reply:
@stefanseifert added a comment that describes the functionality.